tests/tests/helpers.rs

```diff
@@ -71,6 +71,16 @@ pub async fn websocket_server_with_subscription() -> SocketAddr {
 			})
 			.unwrap();
 
+		module
+			.register_subscription("subscribe_noop", "unsubscribe_noop", |_, sink, _| {
```

Comment:
This in example to show an edge-case.

If the subscription request is answered by the server and then the server decides to close the subscription. Then it's not possible to send a response on some request to cancel the subscription.

I don't think it's a clean solution this case. AFAIU, `jsonrpc` has it too.

Thus, I suggest an ugly way of handling this; just send a `notif response` with some weird type as result then rely on the assumption that the client will not add this type on the `Subscription<T>` then it should fail the client calls `Subscription::next`